aws/resource_aws_ecs_service_test.go

```diff
@@ -1350,6 +1380,96 @@ resource "aws_ecs_service" "mongo" {
 `, clusterName, tdName, svcName)
 }
 
+func testAccAWSEcsServiceModifiedWithWait(clusterName, tdName, svcName string) string {
+	return fmt.Sprintf(`
+data "aws_ami" "test" {
+  most_recent = true
+  owners      = ["amazon"]
+
+  filter {
+    name   = "name"
+    values = ["amzn2-ami-ecs-hvm-*"]
+  }
+}
+
+data "aws_availability_zones" "available" {
+  state = "available"
+
+  filter {
+    name   = "opt-in-status"
+    values = ["opt-in-not-required"]
+  }
+}
+
+resource "aws_launch_template" "test" {
+  image_id      = data.aws_ami.test.id
+  instance_type = "t3.micro"
+  name          = %[3]q
+}
+
+resource "aws_autoscaling_group" "test" {
+  availability_zones = data.aws_availability_zones.available.names
+  desired_capacity   = 1
+  max_size           = 1
+  min_size           = 1
+  name               = %[3]q
+
+  launch_template {
+    id = aws_launch_template.test.id
+  }
+
+  tags = [
+    {
+      key                 = "foo"
+      value               = "bar"
+      propagate_at_launch = true
+    },
+  ]
+}
+
```

Comment:
added b/c `DescribeServices` was throwing `was unable to place a task because no container instance met all of its requirements. Reason: No Container Instances were found in your capacity provider` error